projects/taraxa/index.js

```diff
@@ -0,0 +1,33 @@
+const TARAXA_STAKING_CONTRACT = "0x00000000000000000000000000000000000000fe";
+const TARAXA_ECOSYSTEM_FUND_ADDRESS =
+  "0x723304d1357a2334fcf902aa3d232f5139080a1b";
+const TARAXA_FOUNDATION_ADDRESS = "0x723304d1357a2334fcf902aa3d232f5139080a1b";
+
+async function staking(api) {
+  const balance = await api.provider.getBalance(
+    TARAXA_STAKING_CONTRACT,
+    api.block
+  );
+  api.addGasToken(Number(balance) / 10 ** 18);
+}
+
+async function tvl(api) {
+  const balanceEcosystem = await api.provider.getBalance(
+    TARAXA_ECOSYSTEM_FUND_ADDRESS,
```

Comment:
is this your treasury address?

Reply:
The "Foundation" address is the treasury. This is a community fund.

Reply:
Actually both of them count as the project treasury. The foundation tokens are never circulated and the ecosystem fund is for grants (this changes from month to month)

Reply:
okay, we can track this under treasury but not as tvl. making the change now

Reply:
cant count foundation tokens as it is not circulating